packages/client/modules/userDashboard/components/OrgBilling/OrgPlanDrawerContent.tsx

```diff
@@ -0,0 +1,255 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {EnterpriseBenefits, TeamBenefits} from '../../../../utils/constants'
+import {upperFirst} from '../../../../utils/upperFirst'
+import {TierEnum} from '../../../../__generated__/DowngradeToStarterMutation.graphql'
+
+const List = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  overflowY: 'auto',
+  padding: '16px 0px',
+  position: 'relative',
+  width: '100%'
+})
+
+const DrawerContent = styled('div')({
+  backgroundColor: PALETTE.WHITE,
+  display: 'flex',
+  overflow: 'hidden',
+  padding: 16,
+  height: '100vh',
+  flexDirection: 'column'
+})
+
+const Title = styled('span')({
+  fontWeight: 600,
+  fontSize: 20
+})
+
+const Subtitle = styled('span')({
+  fontWeight: 600,
+  paddingBottom: 8
+})
+
+const UL = styled('ul')({
+  margin: 0
+})
+
+const LI = styled('li')<{isBlue?: boolean}>(({isBlue}) => ({
+  fontSize: 16,
+  lineHeight: '28px',
+  color: isBlue ? PALETTE.SKY_500 : PALETTE.SLATE_900,
```

Comment:
<img width="348" alt="image" src="https://user-images.githubusercontent.com/466991/230387413-4ea4750f-71cc-49b3-9c7a-fcb31a80f047.png">

Sometimes it is blue for links, sometimes it black. Do we really need different colors?

Reply:
The designs have bullets in black when there's just text and blue when they're links: 

![image](https://user-images.githubusercontent.com/39854876/230413468-dc2c7214-5d0c-4f20-a9c9-7f2e7243c5ba.png)

The newer content, like the one in your screenshot, has a mix of regular text and links, so I went with black bullets. I think you're probably right that we should be consistent, so I'll just make them all black. 

@acressall let me know if you have any objections 🙏 Here's the update:

![Screenshot 2023-04-06 at 15 43 55](https://user-images.githubusercontent.com/39854876/230414896-cb128e8e-fe0d-411f-84d9-5e210ead1088.png)

Reply:
I like this update! Thanks Igor for catching it and Nick for making the change 🎉